waiter/integration/waiter/kubernetes_scheduler_integration_test.clj

```diff
@@ -337,3 +337,50 @@
                     :interval 15
                     :timeout timeout-secs)))))
         (log/warn "skipping test as INTEGRATION_TEST_BAD_IMAGE is not specified")))))
+
+(deftest ^:parallel ^:integration-fast test-kubernetes-reverse-proxy-sidecar-headers
```

Comment:
We should combine the two tests `test-kubernetes-reverse-proxy-sidecar-headers` and `test-kubernetes-reverse-proxy-sidecar-env` into a single test.

We can then send a canary request and then have separate assertion blocks for the `/request-info` and `/environment` requests.

Reply:
Done